analyses/interaction-plots/01-create-interaction-plots.sh

```diff
@@ -49,6 +49,14 @@ if [ "$ALL" -gt "0" ]; then
 fi
 
 
+# Get FLAG file and add header
```

Comment:
The README for this module should get updated to include a citation for FLAGS, etc.